nerdlets/groundskeeper-nerdlet/AgentStateLoader.js

```diff
@@ -22,15 +22,15 @@ const AGENT_NG_QUERY_WEEK = `query($id: Int!) {
     }
 }`;
 
-export default class AgentStateLoader extends React.Component {
+export default class AgentStateLoader extends React.PureComponent {
```

Comment:
Have you verified that a shallow compare on props and state are always correct about whether to re-render for all of the components? AgentStateLoader should be fine. I'm not entirely confident that the top-level logic always creates new data structures to pass into `props` and `setState` vs. updating existing structures. That could lead `SLAReport` and `Groundskeeper` components to not re-render when they should.